homeassistant/components/wemo/__init__.py

```diff
@@ -212,7 +212,8 @@ async def async_add_unique_device(
             hass.async_create_task(
                 hass.config_entries.async_forward_entry_setups(
                     self._config_entry, platforms_to_load
-                )
+                ),
+                eager_start=True,
```

Comment:
If we start eagerly why don't we await the platform forwarding?

Reply:
This is already running in a task. It could be changed to be awaited if its moved to the end of this block

Reply:
Adjusted in e138d15b329349a53c3d7d2eda86f999d67a00ee